toplevel/byte/topeval.ml

```diff
@@ -214,9 +212,8 @@ let load_compunit ic filename ppf compunit =
   LongString.set code compunit.cu_codesize (Char.chr Opcodes.opRETURN);
   LongString.blit_string "\000\000\000\001\000\000\000" 0
                      code (compunit.cu_codesize + 1) 7;
-  let initial_symtable = Symtable.current_state() in
-  Symtable.patch_object code compunit.cu_reloc;
-  Symtable.update_global_table();
```

Comment:
I don't know if it's a bug or if it's expected, `Symtable.check_global_initialized reloc` is missing here.